apps/dash-summarize/app.py

```diff
@@ -9,18 +9,19 @@
 import torch
 
 device = "cuda" if torch.cuda.is_available() else "cpu"
-# device = "cpu"
 print(f"Device: {device}")
 
 # Load Model
-# pretrained = 'facebook/bart-large-xsum'
-pretrained = "sshleifer/distilbart-xsum-12-3"
+pretrained = "sshleifer/distilbart-xsum-12-6"
```

Comment:
Can you explain the difference between the `sshleifer/distilbart-xsum-12-3` and `sshleifer/distilbart-xsum-12-6` models?

Reply:
12-6 is slightly larger and slower model, but it performs better than 12-3. Since we are reducing the memory usage with `model.half()`, we can use a slightly bigger model :)